src/main/java/net/imagej/omero/DefaultOMEROService.java

```diff
@@ -410,7 +410,7 @@ public long uploadTable(final OMEROCredentials credentials,
 				}
 			}
 			finally {
-				session.close();
+				((DefaultOMEROSession) session).close();
```

Comment:
Is there a reason why you are casting to the Implementation ? If you do this to avoid handeling the IOException, the right way to do this is to override that function in the ``OMEROSession`` interface.